src/order/filter/basic.lean

```diff
@@ -2531,6 +2531,10 @@ lemma tendsto_infi' {f : α → β} {x : ι → filter α} {y : filter β} (i :
   tendsto f (⨅ i, x i) y :=
 hi.mono_left $ infi_le _ _
 
+theorem tendsto_infi_infi {f : α → β} {x : ι → filter α} {y : ι → filter β}
```

Comment:
Is `tendsto_supr_supr` true? I see `tendsto_supr` lower down, but not `tendsto_supr'`.

Reply:
Do we ever need it?

Reply:
I don't know, but for completeness I would like us to have it if it's true